SocialFilter/sections/whitelist.txt

```diff
@@ -12,6 +12,8 @@ designervn.net#@#.fb-share-button
 !#endif
 ! #End Social locker
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/80098
+@@||facebook.com/plugins/likebox.php$domain=indulgexpress.com
```

Comment:
We should block the facebook leftover instead of whitelisting it. And please make a rule for share buttons at the left

Reply:
@Sergey-Lyapin It's a likebox, not share, if I understand correctly.

Reply:
It is social widget, we should block them, you may search `likebox.php` in the filters and find some rules.

Reply:
Ok wait.

Reply:
Fixed.